posts/2017-09-27-designing-the-event-gateway.md

```diff
@@ -1,55 +1,63 @@
 ---
-title: How we approached designing our latest developer tool - the Event Gateway 
-description: Learn how our full approach to designing the Event Gateway, the backbone of your serverless architectures.
+title: How we married passion and practicality to create our latest dev tool - the Event Gateway 
+description: We break down the guiding principles and executable steps we used to design our latest dev tool: the Event Gateway. This is how we towed the line between a passion project and a practical tool for every serverless developer.
 date: 2017-09-27
 layout: Post
 thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/event-gateway-thumbnail.png
 authors:
   - MaciejWinnicki
 ---
 
-[The Event Gateway](https://serverless.com/blog/introducing-serverless-event-gateway/) is our most recent announcement: a new piece of infrastructure that lets developers react to data flows with serverless functions.
+[The Event Gateway](https://serverless.com/blog/introducing-serverless-event-gateway/) is our most recent announcement—and honestly, building it was as much a passion project as it was motivated by an industry need.
 
-See how we did it! This was our design process from planning to execution.
+Serverless development is still fresh, lacking in best practices and tooling. But people use it because the payoffs are worth the pain.
 
-## Design Considerations
+Our job at Serverless is to remove that pain: fight for standardization, think from the top-down about what a serverless application should look like, make its development seamless. The biggest sticking point we see here is that serverless development is fundamentally event-driven development. Every function you deploy to a serverless instance will remain stateless until woken up by an event.
```

Comment:
> Every function you deploy to a serverless instance will remain stateless until woken up by an event.

hmm, it's not common to say "serverless instance". Maybe "Function-as-a-Service provider"? or "FaaS provider"

>will remain stateless

it's always stateless. "idle" is better word here.